src/Cfonb120Reader.php

```diff
@@ -18,49 +18,41 @@
 use Silarhi\Cfonb\Banking\Operation;
 use Silarhi\Cfonb\Banking\OperationDetail;
 use Silarhi\Cfonb\Banking\Statement;
-use Silarhi\Cfonb\Contracts\ParserInterface;
 use Silarhi\Cfonb\Exceptions\ParseException;
 use Silarhi\Cfonb\Parser\Cfonb120\EmptyParser;
 use Silarhi\Cfonb\Parser\Cfonb120\Line01Parser;
 use Silarhi\Cfonb\Parser\Cfonb120\Line04Parser;
 use Silarhi\Cfonb\Parser\Cfonb120\Line05Parser;
 use Silarhi\Cfonb\Parser\Cfonb120\Line07Parser;
-use function strlen;
+use Silarhi\Cfonb\Parser\FileParser;
 
 class Cfonb120Reader
 {
-    /** @var ParserInterface[] */
-    private $parsers;
+    /** @var FileParser */
+    private $fileParser;
 
     public function __construct()
     {
-        $this->parsers = [
+        $this->fileParser = new FileParser(
             new Line01Parser(),
             new Line04Parser(),
             new Line05Parser(),
             new Line07Parser(),
-            new EmptyParser(),
-        ];
+            new EmptyParser()
+        );
     }
 
     /** @return Statement[] */
     public function parse(string $content): array
     {
-        if (!empty($content) && strlen($content) > 120 && false === strpos($content, "\n")) {
-            $content = chunk_split($content, 120, "\n");
-        }
-
         $statementList = [];
-        $lines = explode("\n", $content);
-        $statement = new Statement();
-
-        /** @var Operation|null $lastOperation */
         $lastOperation = null;
-        foreach ($lines as $line) {
-            $result = $this->findSupportedParserForLine($line)->parse($line);
+        $statement = new Statement();
 
+        foreach ($this->fileParser->parse($content, 120) as $result) {
```

Comment:
Could you create a constant to store the 120 length? To also be used at AbstractCfonb120Parser::supports if we merge this class into this one

Reply:
Ok ;)